### ISSUE TITLE
Many errors related to "Pauls Spielwiese" in v.1.0.4

### ISSUE THREAD

**Issue by burque505** [bichlepa/AutoHotFlow#3]
Issue: AHF v.1.0.4 will not start or loads improperly
Steps to reproduce: Compile AutoHotflow.ahk. Run.
Workaround: 

- Edit "main.ahk" and remove references to "Pauls Spielwiese".
- Copy "en.ini" from lang folder to v.1.0.4 AutoHotflow\language folder
- Remove saved flow 1039122644
- Restart

**Comment by bichlepa** [comment]
Thank you for the hint. I have a local package "Pauls Spielwiese" with which I did some tests and this caused some interferences.

I modified "find modules.ahk" and it will include only the default package if a command line parameter is present. I'll use it everytime before I push in future.
I also removed the local flows from the repo that I use for random tests.
The file "language/en.ini" is a merge from all the language files and I removed it from the repo.
Now if "language/en.ini" does not exist, "AutoHotflow.ahk" starts "find modules.ahk" which generates this file.

So, none of those errors should happen again (probably 😅)
